Streetcode/Streetcode.XUnitTest/MediatRTests/Authentication/Register/HandleRegisterTest.cs

```diff
@@ -0,0 +1,288 @@
+﻿// <copyright file="HandleRegisterTest.cs" company="PlaceholderCompany">

```

Comment:
Delete this comments

Reply:
Hi, if we don't add this comment, style analyzer generates following warning `SA 1633`:
![image](https://github.com/ita-social-projects/StreetCode/assets/113620149/e80c06e0-5d60-405b-9ecb-dac5a609efd1)